### PR TITLE
build(deps): Upgrade Cypress and Vue Cli plugins versions.

### PR DESCRIPTION
**Pull request by tajespasarela**
[EX-5417]

Updated VueCli version, and removed unnecessary dependencies

[EX-5417]: https://searchbroker.atlassian.net/browse/EX-5417?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/x-components/package.json [hunk 1]
```diff
@@ -79,9 +79,9 @@
   "devDependencies": {
     "@cucumber/cucumber": "~7.0.0-rc.0",
```
Comment (javieri-empathy) [inline packages/x-components/package.json:80]: This dependency is not needed

File: packages/x-components/package.json [hunk 2]
```diff
@@ -79,9 +79,9 @@
...
     "@cypress/browserify-preprocessor": "~3.0.1",
-    "@cypress/vue": "~2.2.3",
-    "@cypress/webpack-dev-server": "~1.4.0",
-    "@cypress/webpack-preprocessor": "~5.1.1",
+    "@cypress/vue": "~2.2.4",
+    "@cypress/webpack-dev-server": "~1.8.2",
+    "@cypress/webpack-preprocessor": "~5.11.1",
```
Comment (javieri-empathy) [inline packages/x-components/package.json:84]: This dependency is also not needed

File: packages/x-components/package.json [hunk 3]
```diff
@@ -79,9 +79,9 @@
   "devDependencies": {
     "@cucumber/cucumber": "~7.0.0-rc.0",
     "@cypress/browserify-preprocessor": "~3.0.1",
```
Comment (javieri-empathy) [inline packages/x-components/package.json:81]: This dependency is also not needed

File: packages/x-components/package.json [hunk 4]
```diff
@@ -90,10 +90,10 @@
...
     "@types/node": "~16.4.12",
     "@types/testing-library__jest-dom": "~5.9.1",
-    "@vue/cli-plugin-e2e-cypress": "~4.5.16",
-    "@vue/cli-plugin-typescript": "~4.5.16",
-    "@vue/cli-plugin-vuex": "~4.5.16",
-    "@vue/cli-service": "~4.5.16",
+    "@vue/cli-plugin-e2e-cypress": "~4.5.17",
+    "@vue/cli-plugin-typescript": "~4.5.17",
+    "@vue/cli-plugin-vuex": "~4.5.17",
+    "@vue/cli-service": "~4.5.17",
```
Comment (javieri-empathy) [inline packages/x-components/package.json:96]: Shouldn't this be updated to the 5th version?

**Review by javieri-empathy — CHANGES_REQUESTED** [review]

**Review by LuisMartinez15 — APPROVED** [review]
LGTM

**Review by guillei10 — APPROVED** [review]

**Pull request merged**